From: Benjamin Drung <bdrung@debian.org>
Description: Fix typo in vlc.desktop: Name[nn] -> GenericName[nn].
Forwarded: yes

--- a/share/vlc.desktop
+++ b/share/vlc.desktop
@@ -38,7 +38,7 @@
 GenericName[nl]=Mediaspeler
 Comment[nl]=Uw multimediastreams afspelen, opnemen en uitzenden
 Name[nn]=VLC mediespelar
-Name[nn]=Mediespelar
+GenericName[nn]=Mediespelar
 Comment[nn]=Spel av, ta opp og send ut multimedia
 Name[pa]=VLC ਮੀਡਿਆ ਪਲੇਅਰ
 Comment[pa]=ਆਪਣੀ ਮਲਟੀਮੀਡਿਆ ਸਟਰੀਮ ਪੜ੍ਹੋ, ਕੈਪਚਰ ਤੇ ਬਰਾਡਕਾਸਟ ਕਰੋ 
